src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@ public Movie get(Long id) {
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
```

Comment:
Consider changing the return type of the `update` method to `void` or `Movie`. The current implementation returns a Boolean based on an equality check, which might not be reliable or meaningful.